pkg/kubelet/kubelet.sh

```diff
@@ -63,6 +63,24 @@ echo "kubelet.sh: ${await} has arrived" 2>&1
 
 mkdir -p /etc/kubernetes/manifests
 
+if [ -n ${CONTAINERIZED_MOUNTER_HOME} ]; then
+    echo "mount mounter: ${CONTAINERIZED_MOUNTER_HOME}"
+    CONTAINERIZED_MOUNTER_ROOTFS="${CONTAINERIZED_MOUNTER_HOME}"/rootfs
+    mkdir -p /var/lib/kubelet
+    mkdir -p "${CONTAINERIZED_MOUNTER_ROOTFS}"/var/lib/kubelet
+    mkdir -p "${CONTAINERIZED_MOUNTER_ROOTFS}"/proc
+    mkdir -p "${CONTAINERIZED_MOUNTER_ROOTFS}"/dev
+    mkdir -p "${CONTAINERIZED_MOUNTER_ROOTFS}"/etc
+    touch "${CONTAINERIZED_MOUNTER_ROOTFS}"/etc/resolv.conf
+    mount --bind "${CONTAINERIZED_MOUNTER_HOME}" "${CONTAINERIZED_MOUNTER_HOME}"
+    mount -o remount,exec "${CONTAINERIZED_MOUNTER_HOME}"
+    mount --rbind /var/lib/kubelet "${CONTAINERIZED_MOUNTER_ROOTFS}"/var/lib/kubelet
+    mount --bind -o ro /etc/resolv.conf "${CONTAINERIZED_MOUNTER_ROOTFS}"/etc/resolv.conf
+    mount --bind -o ro /proc "${CONTAINERIZED_MOUNTER_ROOTFS}"/proc
+    mount --bind -o ro /dev "${CONTAINERIZED_MOUNTER_ROOTFS}"/dev
```

Comment:
This looks suspiciously like it really ought to be a service defined in the yml. That might mean enhancing the builder tool and/or init to provide a way to reference one container from another, right now `runtime.BindNS.Mnt` looks like it should be usable for that but it actually isn't. I can't find the issue I raised right now though.

Reply:
Here it is https://github.com/moby/tool/issues/152.